services/tokens.js

```diff
@@ -61,7 +62,13 @@ Tokens.createToken = function(config) {
     note: v.string,
     scopes: v.arrayOf(v.string),
     resources: v.arrayOf(v.string),
-    allowedUrls: v.arrayOf(v.string)
+    allowedUrls: v.arrayOf(v.string),
+    allowedApplications: v.arrayOf(
+      v.shape({
+        bundleId: v.string,
+        platform: v.string
+      })
```

Comment:
```suggestion
      v.strictShape({
        bundleId: v.string,
        platform: v.oneOf("Android", "iOS")
      })
```

I wonder do we want to prevent them from dispatching their request if platform is not `Android` or `iOS`?

Reply:
This is intentional as I don't want to encode every piece of validation logic into the SDK itself to make it easier to change in the backend.

Reply:
Yep, potentially we will get more traffic because of Bad Requests but probably fine to gain more flexibility. Not sure what's the strategy here but let's move on. It's good enough for now.